biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/AbstractFeature.java

```diff
@@ -271,13 +271,13 @@ public void setUserObject(Object userObject) {
     }
     
 	@Override
-	public HashMap<String, Qualifier> getQualifiers() {
+	public HashMap<String, ArrayList<Qualifier>> getQualifiers() {
```

Comment:
Fair enough.

The sequencing module depends on Guava too https://github.com/biojava/biojava/blob/master/biojava-sequencing/pom.xml#L15